plugins/techdocs/src/reader/components/Reader.tsx

```diff
@@ -556,20 +556,30 @@ export const useTechDocsReaderDom = (
                 height: 100%;
               }
               .md-sidebar--primary {
-                width: 12.1rem !important;
+                width: 16rem !important;
                 z-index: 200;
                 left: ${
-                  isPinned ? 'calc(-12.1rem + 242px)' : 'calc(-12.1rem + 72px)'
+                  isPinned ? 'calc(-16rem + 242px)' : 'calc(-16rem + 72px)'
```

Comment:
Looks wrong when sidebar is pinned.

```suggestion
                  isPinned ? 'calc(-16rem + 224px)' : 'calc(-16rem + 72px)'
```

Reply:
Hey, good catch!
I pushed a new commit using the sidebarConfig instead, can you review it again, please?